apparmor.d/profiles-a-f/fwupdmgr

```diff
@@ -37,12 +60,12 @@ profile fwupdmgr @{exec_path} flags=(attach_disconnected,complain) {
   owner /var/cache/private/fwupdmgr/fwupd/lvfs-metadata.xml.gz{,.asc} rw,
 
   owner @{user_cache_dirs}/ rw,
+        @{user_cache_dirs}/dconf/user rw,
```

Comment:
Remove, already present in `abstractions/dconf-write`

Reply:
Interesting, because it did show up in my audit log so it wasn't permitted. Will look into it.

Reply:
dconf-write has read only for `dconf/user`, this adds write.

Should write be part of the abstraction?

```
# apparmor.d - Full set of apparmor profiles
# Copyright (C) 2022 Alexandre Pujol <alexandre@pujol.io>
# SPDX-License-Identifier: GPL-2.0-only

# Permissions for querying dconf settings with write access; use the dconf
# abstraction first, and dconf-write only for specific application's profile.

  /etc/dconf/** r,

  owner @{user_config_dirs}/dconf/user r,

  owner @{run}/user/@{uid}/dconf/ rw,
  owner @{run}/user/@{uid}/dconf/user rw,

  include if exists <abstractions/dconf-write.d>
```

Reply:
First time I see this need. So let's keep it there for now.